.github/workflows/main.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on:
+  push:         {branches: [master]}
+  pull_request: {branches: [master]}
```

Comment:
Out of curiosity, why this YAML syntax with explicit dictionary?

Reply:
The only reason was to experiment with writing in a more concise format (one line with three words instead of 3 lines with one word). I have no problem switching it to the more typical format.

Reply:
Yes, please 🙏